lib/galaxy/tools/parameters/wrapped_json.py

```diff
@@ -88,7 +92,7 @@ def _data_input_to_path(v):
     elif input_type == "boolean":
         json_value = _cast_if_not_none(value_wrapper, bool)
     elif input_type == "select":
-        if input.multiple:
+        if input.multiple and profile >= '20.05':
             json_value = [_ for _ in _cast_if_not_none(value_wrapper.value, list)]
         else:
             json_value = _cast_if_not_none(value_wrapper.value, str)
```

Comment:
```suggestion
            json_value = _cast_if_not_none(value_wrapper, str)
```
This fixed https://github.com/galaxyproject/galaxy/issues/9773  for me, though not sure if that has some different side-effects

Reply:
Thanks @AndreasSko your suggestion is correct, casting `value_wrapper` was the implementation before https://github.com/galaxyproject/galaxy/pull/9279